### PR TITLE
fix iot guide with new template files path

### PR DESCRIPTION
**Pull request by jbtrystram**
Files were moved recently but the guide was not updated

### CODE REVIEW

File: iot/examples/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
   oc apply -f templates/build/enmasse-latest/install/components/example-plans
   
   oc apply -f templates/build/enmasse-latest/install/preview-bundles/iot
-  oc apply -f iot/examples/iot-config.yaml
+  oc apply -f templates/iot/examples/iot-config.yaml
```
Comment (dejanb) [inline iot/examples/README.md]: The patch should be "install/components/iot/examples/" for all these resources

Comment (jbtrystram) [reply iot/examples/README.md]: why was it moved there then ? 
https://github.com/EnMasseProject/enmasse/blob/master/templates/iot/examples/iot-config.yaml

**Comment by jbtrystram** [comment]
@dejanb WDT ?

**Comment by dejanb** [comment]
@jbtrystram you don't need "templates/build/enmasse-latest/", you should execute commands from that folder (as that's how release will look like and what doc should cover)

File: iot/examples/README.md (hunk 1 above)
Comment (dejanb) [reply iot/examples/README.md]: That's the source of templates ...

**Pull request closed without merging**